packages/client/src/types.ts

```diff
@@ -86,6 +84,15 @@ export type setTopicSessionDataFn = (
   topicData: null | Partial<TopicData>
 ) => void;
 
+/**
+ * PreQuestions enable us to configure custom questions before the IMTR questions. We will use these questions to customise the Outcome Section.
+ *
+ * Direct importing and including components does not work because the hooks lose context.
+ */
+export enum PreQuestionComponent {
+  MULTIPLE_CHECKERS, // Corresponds to PreQuestionMultipleCheckers.tsx
```

Comment:
imtr client should not know about `PreQuestionMultipleCheckers.tsx` Maybe remove the comment.

Reply:
This is types in client and not in imtr-client...